### PR TITLE
Misc bug fixes

### PR DESCRIPTION
**Pull request by garethyr**
Data followup to https://github.com/cortex-command-community/Cortex-Command-Community-Project-Source/pull/434

### CODE REVIEW

File: Base.rte/Activities.ini [hunk 1]
```diff
@@ -11,6 +11,20 @@ AddActivity = GATutorial
 	Team1Funds = 1500
 	CPUTeam = 1
 	MaxPlayerSupport = 1
+	
+
+AddActivity = GAScripted
+	PresetName = Test Activity
```
Comment (MaximDude) [inline Base.rte/Activities.ini:17]: I know this is convenient but move it back to the end.
Eventually:tm: it won't even be part of vanilla.

File: Base.rte/Actors/Brains/Case/BrainCase.ini [hunk 2]
```diff
@@ -94,6 +94,7 @@ AddActor = Actor
 			FilePath = Base.rte/Actors/Brains/Case/Sounds/BrainPop.flac
 	Team = 0
 	Health = 100
+	Robotic = 1
```
Comment (MaximDude) [inline Base.rte/Actors/Brains/Case/BrainCase.ini:97]: Should probably be changed to `Synthetic`, but doesn't have to be done now (to maximize mod breakage in the future:tm: lelelelel)

Also, wouldn't the brain be both?

File: Base.rte/GUIs/MultiplayerGameGUI.ini [hunk 3]
```diff
@@ -9,6 +9,7 @@ Visible = True
 Enabled = True
 Name = base
 Anchor = Left, Top
+ToolTip =
```
Comment (MaximDude) [inline Base.rte/GUIs/MultiplayerGameGUI.ini:12]: Add `None` to empty tooltips.
I'm pretty sure I added a garbage fix to deal with them sometimes being shitty but better be safe (even though they don't show here but whatever. Consistency:tm:).

**Review by MaximDude — CHANGES_REQUESTED** [review]

File: Base.rte/GUIs/MultiplayerGameGUI.ini (hunk 3 above)
Comment (garethyr) [reply Base.rte/GUIs/MultiplayerGameGUI.ini:12]: Will do. Does gui maker not spit out None?

File: Base.rte/Actors/Brains/Case/BrainCase.ini (hunk 2 above)
Comment (garethyr) [reply Base.rte/Actors/Brains/Case/BrainCase.ini:97]: Hmmm, I dunno if brain counts as organic? I guess I kinda figured it's hermetically sealed so it doesn't really count, but techincally there is organic shit in there so maybe.

As for renaming this, I guess I can. You should post it somewhere so people can bitch at you about it though :D

File: Base.rte/Activities.ini (hunk 1 above)
Comment (garethyr) [reply Base.rte/Activities.ini:17]: Booooooooo

Fine

File: Base.rte/Actors/Brains/Case/BrainCase.ini (hunk 2 above)
Comment (MaximDude) [reply Base.rte/Actors/Brains/Case/BrainCase.ini:97]: Meh, can be both, can be either, can be neither. \_/¯( ͜。 ͡ʖ ͜。)¯\\\_
And you can always blame me for whatever. lel

File: Base.rte/Activities.ini (hunk 1 above)
Comment (MaximDude) [reply Base.rte/Activities.ini]: Fine shmine it's still not back where it was. Now it's stuck in the middle of other activities. 😠

**Review by MaximDude — APPROVED** [review]

**Review by MaximDude — APPROVED** [review]

**Pull request merged**